src/javascript/app/pages/user/account/settings/self_exclusion.js

```diff
@@ -89,6 +89,8 @@ const SelfExclusion = (() => {
                     if (key === 'exclude_until') {
                         setDateTimePicker(exclude_until_id, value);
                         $form.find('label[for="exclude_until"]').text('Excluded from the website until');
+                        $('#ukgc_requirement_notice').setVisibility(1);
+                        $('#ukgc_gamstop').setVisibility(is_gamstop_client);
```

Comment:
same here, can we decide visibility of the sentence before we make the paragraph visible?